projects/observability/src/shared/components/top-n/top-n-chart.component.scss

```diff
@@ -29,6 +28,7 @@
     border-radius: 3px;
     height: 6px;
     background-color: $gray-2;
+    cursor: pointer;
```

Comment:
Progress bar isn't clickable, why is it a pointer?

Reply:
I thought it's going to show some tooltip on it, since we removed the %age

Reply:
Don't think so, but tooltip shouldn't be pointer regardless since it's still not clickable

Reply:
Yeah, I missed to remove this. I had added it for a reason. nvm. Will update this